Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkController.cs

```diff
@@ -109,6 +110,39 @@ ThemeColorController FindThemeColorController()
             return controllerTransform.GetComponent<ThemeColorController>();
         }
 
+        private void SetTransformSync(Transform transf, ControllerSyncMode syncMode, ControllerSyncMode targetMode)
+        {
+
+            #if UDONSHARP
+
+            transf.GetComponent<UdonBehaviour>().enabled = syncMode < targetMode;
+
+            #endif
+
+        }
+
+        void SetControllerSyncMode(ControllerSyncMode syncMode)
```

Comment:
This should probably be public so user scripts can call it.

Reply:
SetControllerSyncMode has been made public